src/styles/mo-components/_buttons.scss

```diff
@@ -1,17 +1,45 @@
 button {
+  appearance: none;
+  display: inline-flex;
+  justify-content: center;
+  align-items: center;
+  border: 0;
+  border-radius: 0;
+  cursor: pointer;
+  text-transform: uppercase;
+  background-image: none;
+  box-shadow: none;
+
+  transition: all .3s;
+
+  svg {
+    max-width: 100%;
+    max-height: 100%;
+    transition: all .3s;
+  }
+
   &.btn {
-    font-size: 1em;
-    padding: .375rem .75rem;
+    @include font(roboto-condensed-bold);
```

Comment:
Need this font family to apply more generally to the `button` element to avoid regressions elsewhere (I see buttons with browser default font on some comp pages when I build this branch). Otherwise this looks good!